sendKindle.py

```diff
@@ -169,8 +169,15 @@ def send_mail(self):
 
         # send email
         try:
-            mail_server = smtplib.SMTP_SSL(host=self.smtp_server,
-                                          port=self.smtp_port)
+            if self.smtp_port == 465:
+                mail_server = smtplib.SMTP_SSL(host=self.smtp_server,
+                        port=self.smtp_port)
+            elif self.smtp_port == 587:
+                mail_server = smtplib.SMTP(host=self.smtp_server, port=self.smtp_port)
+                mail_server.starttls()
+            else:
+                raise Exception('port {} not supported'.format(self.smtp_port))
```

Comment:
Why would you restrict the port to just two allowed values? Why not use something like this instead?
```python
if self.smtp_port == 587:
                 mail_server = smtplib.SMTP(...)
                 mail_server.starttls()
else:
                 mail_server = smtplib.SMTP_SSL(host=self.smtp_server,
                        port=self.smtp_port) 
```

Reply:
Good point, this version is better.